scripts/staking-router-v2/sr-v2-deploy-holesky.ts

```diff
@@ -0,0 +1,235 @@
+import { ethers, run } from "hardhat";
+
+import { DepositSecurityModule, DepositSecurityModule__factory } from "typechain-types";
+
+import {
+  deployImplementation,
+  deployWithoutProxy,
+  loadContract,
+  log,
+  persistNetworkState,
+  readNetworkState,
+  Sk,
+} from "lib";
+
+function getEnvVariable(name: string, defaultValue?: string) {
+  const value = process.env[name];
+  if (value === undefined) {
+    if (defaultValue === undefined) {
+      throw new Error(`Env variable ${name} must be set`);
+    }
+    return defaultValue;
+  } else {
+    log(`Using env variable ${name}=${value}`);
+    return value;
+  }
+}
+
+async function main() {
+  // DSM args
+  const PAUSE_INTENT_VALIDITY_PERIOD_BLOCKS = 7200;
+  const MAX_OPERATORS_PER_UNVETTING = 20;
```

Comment:
200 looks like an argument value for this parameter. The study was done by @mkurayan, he can provide more details

```suggestion
  const MAX_OPERATORS_PER_UNVETTING = 200;
```